src/plugins/wizard/server/plugin.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  PluginInitializerContext,
+  CoreSetup,
+  CoreStart,
+  Plugin,
+  Logger,
+} from '../../../core/server';
+
+import { WizardPluginSetup, WizardPluginStart } from './types';
+import { capabilitiesProvider } from './capabilities_provider';
+import { defineRoutes } from './routes';
+import { wizardSavedObjectType } from './saved_objects';
+
+export class WizardPlugin implements Plugin<WizardPluginSetup, WizardPluginStart> {
+  private readonly logger: Logger;
+
+  constructor(initializerContext: PluginInitializerContext) {
+    this.logger = initializerContext.logger.get();
+  }
+
+  public setup({ capabilities, http, savedObjects }: CoreSetup) {
+    this.logger.debug('wizard: Setup');
+    const router = http.createRouter();
+
+    // Register server side APIs
+    defineRoutes(router);
+
```

Comment:
I think so too. Just left it in here since this too came as a part of the scaffolding. Since we dont have docs outlining how to add routes correctly, I figured that there was no harm leaving this in. `defineRoutes` does not actually register any routes though. We just have commented placeholder code there. I'm okay with it either way.